free-1.md

```diff
@@ -10,34 +13,75 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Legal writing is both notoriously complex and requires a lot of context to interpret.
+The users of the language would probably be lawyers, judges, lawmakers or reporters who wanted
+to dig into the context, history and use of various laws, definitions, rulings and interpretations.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Laws, rulings, etc. already need to be documented, and need to be accessed in a document form.
+A DSL is appropriate in that it can also supersede that use case, because it contains both
+the document form and metadata that is eminently useful for people browsing and researching
+the documents.  Additionally, it is helpful in that it could require a minimum of modification
+from the original documents to become syntactically valid (if not semantically useful).
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I came up with the project because I thought of the name "LegalEase", which has of course
+been invented a million times already.  It's exciting because I imagine that legal information
+comprises a hideously large amount of information, and it really needs a sensible means of
+managing that information via computer. (Not interpreting: law isn't black and white. 
+Just connections.)
+Especially if it could eventually be brought to the
+point of making it easier for laypeople to understand, or even point out loopholes and such.
+That's massively beyond the scope, but first there needs to be a standardized, fairly easy,
+not-going-to-be-buggy O(n) way to put the information into a format that the computer can
+manage.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Organizing and connecting legal documentation.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+A user would most likely interact with the language by using a GUI interface,
+because they are not going to be programmers-at-heart.
+However, the basic level would just be adding specific kinds of metadata,
+probably in a format similar to XML.
```

Comment:
If it were built in as a plutin to a standard word processor or whatever most legislatures use it would be most useful